docs/Documentation.md

```diff
@@ -147,6 +147,7 @@ fsharp_disable_elmish_syntax=false
 fsharp_keep_indent_in_branch=false
 fsharp_blank_lines_around_nested_multiline_expressions=true
 fsharp_bar_before_discriminated_union_declaration=false
+fsharp_keep_max_blank_lines=100
```

Comment:
Maybe go for `fsharp_keep_max_number_of_blank_lines`.
We used `number_of_` before. But I don't have a strong opinion on it.
`fsharp_keep_max_blank_lines` would work for me as well.